### ISSUE TITLE
Live streaming: first user connects, all subsequent users fail to connect

### ISSUE THREAD

**Issue by walt93** [WWBN/AVideo#5614]
**Describe the question**
Testing live streaming from OBS
Machine 1 is streaming
Machine 2 connects, plays
Machine 2 opens additional browser windows, these just spin
Machine 3 connects, just spins
Phone 1 connects, just spins

**Screenshots**
![Screen Shot 2021-08-24 at 7 09 02 PM](https://user-images.githubusercontent.com/11711050/130716646-a37b290a-5732-4adf-aff3-deb4cab13d5f.png)

**Comment by DanielnetoDotCom** [comment]
maybe you or your server does not have enough bandwidth for that.

if you have a good CPU you may want to try the adaptive mode

https://github.com/WWBN/AVideo/wiki/Adaptive-Bitrates-on-Livestream

**Comment by walt93** [comment]
How would I be able to tell? I’m using CDN, and can see the Nginx process for one 720p stream is at 3-5% CPU. There are very few users on the server. 

If I have three browsers on computer B only the first browser to be launched will connect. Yet that browser can open many windows that all connect. They are separate sessions and their audio is out of sync. 

Tl;dr I can connect multiple streams from one browser. Not a bandwidth issue.

**Comment by DanielnetoDotCom** [comment]
> If I have three browsers on computer B only the first browser to be launched will connect. Yet that browser can open many windows that all connect. 

They should be the same session (if they are the same browser)

> They are separate sessions and their audio is out of sync.

this is probably something in your RTMP publisher, try to google for that.
